code/datums/wounds/types/dislocations.dm

```diff
@@ -45,6 +45,8 @@
 	. = ..()
 	REMOVE_TRAIT(affected, TRAIT_FINGERLESS, "[type]")
 	REMOVE_TRAIT(affected, TRAIT_BRITTLE, "[type]")
+	if(!affected.owner)
```

Comment:
Why would that even be null?

Reply:
Best guess is, probably a fail-safe if dislocations don't work proper, for example I am pretty sure the brittle trait is applied when a limb is badly damaged, to make hacking it off easier.